doc/conf.py

```diff
@@ -24,16 +24,16 @@
 # If your extensions are in another directory, add it here. If the directory
 # is relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
-sys.path.append(os.path.abspath('.'))
```

Comment:
There's a lot of unnecessary churn here; you really should configure `black`/`pre-commit`/whatever to only work on the diff you actually touched.

Reply:
I'll strip out the churn.